_posts/2023-03-29-kubeflow-1.7-release.md

```diff
@@ -0,0 +1,190 @@
+   </td>
+   <td><strong>Tekton</strong>
+   </td>
+   <td><strong>Oidc-authservice</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>1.25 / <strong>1.24</strong>
+   </td>
+   <td>1.16
+   </td>
+   <td>1.8.1
+   </td>
+   <td>3.2 or 5.0
+   </td>
+   <td>1.10.1
+   </td>
+   <td>2.31.2
+   </td>
+   <td>3.3.8
+   </td>
+   <td>1.5
+   </td>
+   <td><code>e236439</code>
+   </td>
+  </tr>
+</table>
+
+<br />
+
+Another valuable platform enhancement is the support of additional processor architectures including IBM’s Power ([#6684](https://github.com/kubeflow/kubeflow/issues/6684)). This effort provides the foundation to add other processor types as well.
+
+The 1.7 documentation includes overall [installation instructions](https://github.com/kubeflow/manifests#kubeflow-manifests) from the Manifest Working Group, and detailed feature reviews from each Kubeflow working group.  Most of the working groups have broken their changelogs into subsections that highlight core features, UI enhancements, miscellaneous updates, bug fixes and breaking changes.
+
+Working Group Changelogs including breaking changes
+
+* [Notebooks](https://github.com/kubeflow/kubeflow/releases) 
+* [Training Operator](https://github.com/kubeflow/training-operator/blob/master/CHANGELOG.md)
+* [Katib](https://github.com/kubeflow/katib/blob/master/CHANGELOG.md)  
+* [Pipelines](https://github.com/kubeflow/pipelines/blob/master/CHANGELOG.md)  
+* [KServe](https://github.com/kserve/kserve/releases/)
+
+The community has continued its work to identify core components and add-ons.  Significant enhancements in add-ons include the continued integration with KServe’s v.10 release, as well as a new serving option from BentoML. The BentoML team has done a tremendous job in supporting the Kubeflow community in 1.7 and their documentation is excellent.  Details on BentoML are available [here](https://modelserving.com/blog/streamline-production-ml-with-bentoml-and-kubeflow).  
+
+
+## What’s next
+
+The community continues to see a large increase in activity since the [announcement](https://blog.kubeflow.org/kubeflow-applied-cncf-incubating/) that Kubeflow will be donated to the CNCF by Google. The community holds regular meetings to review progress on the checklist items needed for the CNCF due diligence ([meeting notes](https://docs.google.com/document/d/1HXAl6ew5ZUgQaAnEHS1qEPxA5puUz2knUwXOZHU39sA/edit?userstoinvite=mchmarny@google.com&actionButton=1)).
+
+During the 1.7 release cycle, the community formed a Security Team, which is working to improve the security profile of Kubeflow components and their dependencies. The Security Team has completed these three deliveries:
+
+<br />
+
+1. Set-up a github directory, slack channel and regular meeting schedule with [notes](https://docs.google.com/document/d/10_uYpaFWvY2NyqGBPd0dbTt1RumX8MedNcGzKSdtSOY/edit?pli=1#)
+2. Created an [inventory](https://pastebin.ubuntu.com/p/4nMrk4SXjm/) of container images used by Kubeflow
```

Comment:
This is not the correct list of the images used by Kubeflow, there is a PR open to adding the correct list: https://github.com/kubeflow/manifests/pull/2419/files

Reply:
Oh! This is a lot better, lets try and get this into manifests before the release

Reply:
I've updated the link and I'll make sure this PR gets merged before this blog post.